README.md

```diff
@@ -8,12 +8,13 @@ Please check the [Sample App](https://github.com/metaplex-foundation/metaplex-io
 
 ## Installation
 
-From Xcode 11, you can use [Swift Package Manager](https://swift.org/package-manager/) to add Solana.swift to your project.
+From Xcode 14, you can use [Swift Package Manager](https://swift.org/package-manager/) to add Solana.swift to your project.
```

Comment:
So this is actually an error from the previous documentation. While technically yes, we are adding `Solana.swift` the our project, we are adding more than that. So let's replace `Solana.swift` to `Metaplex-iOS`

Reply:
Got it, thanks for explaining that! I've made the necessary change :))